yarn-project/sequencer-client/src/publisher/l1-publisher.test.ts

```diff
@@ -7,12 +7,15 @@ import { L1Publisher, L1PublisherTxSender, MinimalTransactionReceipt } from './l
 
 describe('L1Publisher', () => {
   let txSender: MockProxy<L1PublisherTxSender>;
-  let txHash: string;
-  let txReceipt: MinimalTransactionReceipt;
+  let publishTxHash: string;
+  let processTxHash: string;
+  let processTxReceipt: MinimalTransactionReceipt;
+  let publishTxReceipt: MinimalTransactionReceipt;
   let l2Block: L2Block;
 
   let header: Buffer;
   let archive: Buffer;
+  let txsHash: Buffer;
```

Comment:
Not an actual comment, but thought you would enjoy this: 
https://github.com/ethereum/go-ethereum/blob/830f3c764c21f0d314ae0f7e60d6dd581dc540ce/core/types/block.go#L70